packages/react/src/components/SuiteHeader/IdleLogoutConfirmationModal/IdleLogoutConfirmationModal.jsx

```diff
@@ -3,9 +3,9 @@ import PropTypes from 'prop-types';
 import classnames from 'classnames';
 
 import { settings } from '../../../constants/Settings';
-import { Modal } from '../../../index';
-import SuiteHeader from '../SuiteHeader';
+import { Modal } from '../../Modal';
 import IdleTimer from '../util/IdleTimer';
+import { SUITE_HEADER_ROUTE_TYPES } from '../suiteHeaderConstants';
```

Comment:
ROUTE_TYPES needed to be split into their own file for importing to prevent dependency cycles.